.github/ISSUE_TEMPLATE.md

```diff
@@ -18,6 +18,8 @@
 ```
 insert any relevant code here else remove this section
 ```
+**Screenshot:**
```

Comment:
Screenshot is optional and can be accommodated in the Current and expected behaviour section of the templates. 
User will have to manually delete this line in case there isn't a screenshot.  Rest looks good :)

Reply:
@CosmicCoder96 I'll make the changes.Thanks!